Amazon.IonDotnet/Internals/Text/SystemTextReader.cs

```diff
@@ -526,7 +526,11 @@ public override string[] GetTypeAnnotations()
             for (int index = 0; index < _annotations.Count; index++)
             {
                 SymbolToken symbolToken = _annotations[index];
-                if (symbolToken.Text is null && symbolToken.ImportLocation != default)
+                if (symbolToken.ImportLocation == default)
```

Comment:
This doesn't look right, since most symbols with known text should have a default ImportLocation. Are we missing test coverage for this?

I think it was almost right before, I just wanted it generalized so that the condition for symbols with unknown text is `ImportLocation == default`, not `Sid == 0`.

Reply:
Reverted change, updated to Sid==0

Reply:
Created issue: [symbol with known text has non-default ImportLocation](https://github.com/amzn/ion-dotnet/issues/98)